sympy/printing/latex.py

```diff
@@ -2058,6 +2058,10 @@ def _print_primeomega(self, expr, exp=None):
                     self._print(exp))
         return r'\Omega\left(%s\right)' % self._print(expr.args[0])
 
+    def _print_Quantity(self, expr):
+        if str(expr) == 'degree':
```

Comment:
Use self.name. Don't rely on the str printer here.

Reply:
`Latexprinter` class has no attribute `name` . So using `self.name` throws `AttributeError` . Please point out if I am getting something wrong.

Reply:
I think he meant `expr.name`.

Reply:
Type of `expr.name` is Symbol. 
Therefore we have to add check like 
```
if expr.name == Symbol('degree'):
    return r"^\circ"
``` 
Is this format alright or is there something that I am missing.

Reply:
You can do that or you can do `expr.name.name`.